datadog/resource_datadog_dashboard.go

```diff
@@ -4917,6 +4933,156 @@ func buildTerraformServiceLevelObjectiveDefinition(datadogDefinition datadogV1.S
 	return terraformDefinition
 }
 
+//
+// SLO (Service Level Objective) List Widget Definition helpers
+//
+
+func getSloListDefinitionSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"request": {
+			Description: "A nested block describing the request to use when displaying the widget. Exactly one `request` block is allowed.",
+			Type:        schema.TypeList,
+			Optional:    true,
```

Comment:
Should this be required instead (basing it on build definition)?

Reply:
Good idea, done